source/interact-data/crud.txt

```diff
@@ -0,0 +1,271 @@
+.. _django-crud:
+
+=======================
+Perform CRUD Operations
+=======================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: insert, modify, read, write, code example
+
+Overview
+---------
+
+In this guide, you can learn how to use {+django-odm+} to run
+create, read, update, and delete (CRUD) operations on your MongoDB
+collection.
+
+You can use methods provided by the Django ``QuerySet`` API to run
+CRUD operations. To run these operations, you can call ``QuerySet`` methods
+on your model's ``Manager``. The ``Manager`` class handles database
+operations and allows you to interact with your MongoDB data by referencing
+Django models. By default, Django adds a ``Manager`` named ``objects``
+to every model class. 
+
+This guide shows how to use the following ``QuerySet`` methods:
+
+- :ref:`create() <django-crud-insert>`
+- :ref:`filter() and get() <django-crud-read>`
+- :ref:`update() <django-crud-modify>`
+- :ref:`delete() <django-crud-delete>`
+
+.. tip::
+
+    To learn more about Django's ``QuerySet`` API, see
+    `QuerySet API reference <https://docs.djangoproject.com/en/5.1/ref/models/querysets/>`__
+    in the Django documentation.
+
+You can also use the Django admin site to edit your models
+and their corresponding collections on a web interface. For
+more information, see `The Django admin site <https://docs.djangoproject.com/en/5.1/ref/contrib/admin/>`__
+in the Django documentation.
+
+Sample Data
+~~~~~~~~~~~
+
+The examples in this guide use the ``Movie`` model, which represents
+the ``sample_mflix.movies`` collection from the :atlas:`Atlas sample datasets </sample-data>`.
+The ``Movie`` model class has the following definition:
+
+.. code-block:: python
```

Comment:
Q: guessing that when these files are moved to the correct repo that you'll move these into files?

Reply:
yes!